app/javascript/components/editor/TestSummary.tsx

```diff
@@ -44,19 +47,19 @@ export function TestSummary({
         {isPresent(test.testCode) ? (
           <div className="--info">
             <h3>Code Run</h3>
-            <pre>{test.testCode}</pre>
+            <pre dangerouslySetInnerHTML={{ __html: test.testCodeHtml }}></pre>
```

Comment:
I've also rendered the test code as HTML, as I could imagine that using syntax highlighting colors there might be really beneficial for tracks that can do that.